dataset_analysis/README.md

```diff
@@ -0,0 +1,26 @@
+# Installation
+
+To install the required libraries run :
+
+~~~
+pip install -r requirements_dataframe.txt
+~~~
+
+It is also required to install SpinalCordToolbox 6.0 :
```

Comment:
Is SCT really required? It seems you are using `image.py` from ivadomed/utils.

Reply:
Yes because I use sct_analyse_lesion

Reply:
Okay!